doc/sources/installation/installation-windows.rst

```diff
@@ -30,8 +30,13 @@ terminal that has Python available.
 
      python -m pip install --upgrade pip wheel setuptools virtualenv
 
-   **Optional (but recommened!)**: Create a new `virtual environment <https://virtualenv.pypa.io/en/latest/>`_
-   for your Kivy project:
+   **Optional**: Create a new `virtual environment <https://virtualenv.pypa.io/en/latest/>`_
+   for your Kivy project.
+   
+   .. warning::
```

Comment:
Almost there. I'd remove the warning paragraph and write it like you had it before, but as `Optional **(but strongly recommended)**: ...` Followed by a one sentence note that "a virtualenv will prevent possible installation conflicts with other python versions and with other packages".

Because the paragraph gives a lot of extra information which is not needed by typical users and it's best to keep extra info to a minimum so people don't get sidetracked or confused when installing.

Reply:
Yes, that is true. I changed it.